### PR TITLE
doc: clarify two fields of configuring the languages

### PR DESCRIPTION
**Pull request by HeavySnowJakarta**
improves the doc

### CODE REVIEW

**Comment by HeavySnowJakarta** [comment]
attention i've forgot about what happened about `crates/live_kit_server/protocol` at that time. maybe have to give up this commit and make a new one

**Comment by HeavySnowJakarta** [comment]
@cla-bot check

**Comment by maxdeviant** [comment]
> attention i've forgot about what happened about `crates/live_kit_server/protocol` at that time. maybe have to give up this commit and make a new one

I removed it from this PR.

File: docs/src/configuring-languages.md [hunk 1]
```diff
@@ -57,7 +57,14 @@ You can customize a wide range of settings for each language, including:
 - `preferred_line_length`: The recommended maximum line length
 - `soft_wrap`: How to wrap long lines of code
 
-These settings allow you to maintain specific coding styles across different languages and projects.
+These settings allow you to maintain specific coding styles across different languages and projects. Specifically, the field `soft_wrap` can be:
+
+- `none`: Do not soft wrap.
```
Comment (maxdeviant) [inline docs/src/configuring-languages.md:62]: We already have the values of `soft_wrap` documented here: https://github.com/zed-industries/zed/blob/cf0a8a7a1a47cfda5a48348d8309a00a737f2b1e/docs/src/configuring-zed.md#L1182-L1193

I don't think it makes sense to duplicate them here as well.

**Comment by maxdeviant** [comment]
Thank you for putting together this PR!

I don't think it makes sense to duplicate the setting information across multiple pages.

In https://github.com/zed-industries/zed/pull/17082 we now link to the definition of the referenced settings.

**Comment by HeavySnowJakarta** [comment]
> In #17082 we now link to the definition of the referenced settings.

well that seems to be the best solution

**Pull request closed without merging**